packages/docs/src/scripts/components/UsaBanner.jsx

```diff
@@ -12,7 +12,7 @@ const UsaBanner = function(props) {
         <div className="ds-u-display--flex ds-u-flex-direction--row ds-u-align-items--start ds-u-sm-align-items--center">
           <img
             className="c-usa-banner__header-flag"
-            src="/public/images/us_flag_small.png"
+            src="./public/images/us_flag_small.png"
```

Comment:
oh wow thats all it took? thats great! i assumed it was some sort of build process issue

Reply:
Yeah I think so, I was doing some testing in the browser but don't think any of the build would muck up the path